server/src/resolvers/Query.js

```diff
@@ -34,7 +34,7 @@ const Query = {
   },
 
   me(parent, args, ctx, info) {
-    const id = getUserId(ctx);
+    const id = getUserIdOptional(ctx);
```

Comment:
workaround for `refetchQueries` issue, a better approach would involve addressing how errors are handled by the server